var/spack/repos/builtin/packages/cube/package.py

```diff
@@ -46,16 +51,19 @@ class Cube(Package):
     variant('gui', default=False, description='Build CUBE GUI')
 
     depends_on('zlib')
-    depends_on('qt@4.6:', when='+gui')
 
-    def install(self, spec, prefix):
-        configure_args = ["--prefix=%s" % prefix,
-                          "--without-paraver"]
```

Comment:
Builds and works fine without `--without-paraver`. Doesn't introduce any additional dependencies either. TBH I don't know why it was there in the first place.